__tests__/app.test.js

```diff
@@ -121,3 +126,49 @@ describe("api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 returns an array of comments for specific article Id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then((response) => {
+        const comments = response.body.comments;
+        expect(comments.length).toBe(2);
+        comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
```

Comment:
if all of these comments come from id 5 I would prefer it if you look at the actual value of comment_id and make sure it's always 5